src/SignalR/clients/cpp/src/signalrclient/default_http_client.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+#include "default_http_client.h"
+
+namespace signalr
+{
+    void default_http_client::send(std::string url, http_request request, std::function<void(http_response, std::exception_ptr)> callback)
+    {
+        web::http::method method;
+        if (request.method == http_method::GET)
+        {
+            method = U("GET");
+        }
+        else if (request.method == http_method::POST)
+        {
+            method = U("POST");
+        }
```

Comment:
nit: fail properly in else case in case a new method is added, someone does an unsafe cast, etc...